theme/static/style/style.css

```diff
@@ -174,12 +174,12 @@ nav button {
 }
 
 /* Article styles */
-.article h1, .community h1 { margin-bottom: 3rem; }
+.article h1, .community section h1 { margin-bottom: 3rem; }
```

Comment:
Why do you need the extra `section` selector?

Reply:
The extra section selector is used because we want to apply the styling only to the inner section tag and not to the navigation and footer sections.